src/components/AutoComplete/AutoComplete.tsx

```diff
@@ -30,22 +39,29 @@ interface Props {
   startAdornment?: JSX.Element;
   disableUnderline?: boolean;
   onChange: (event: React.FormEvent<HTMLInputElement>, params: ChangeEvent) => void;
-  onSuggestionsFetch: ({ value: string }) => Promise<void>;
+  onSuggestionsFetch: SuggestionsFetchRequested;
   onCleanSuggestions?: () => void;
   onClick?: (event: React.FormEvent<HTMLInputElement>, data: SuggestionSelectedEventData<unknown>) => void;
   onKeyDown?: (event: KeyboardEvent<HTMLInputElement>) => void;
   onBlur?: (event: React.FormEvent<HTMLInputElement>) => void;
 }
 
+interface Suggestion {
+  name: string;
+}
+
+type CustomInputProps = Pick<Props, 'disableUnderline' | 'startAdornment'>;
+
 /* eslint-disable react/jsx-sort-props  */
 /* eslint-disable verdaccio/jsx-spread */
-const renderInputComponent = (inputProps): JSX.Element => {
+const renderInputComponent: RenderInputComponent<Suggestion> = inputProps => {
+  // @ts-ignore
```

Comment:
this is the new ts-ignore, since there is currently no way to type `startAdornment` & `disableUnderline`

Reply:
Ok, we can find a solution later for this